src/IB/IIMethod.cpp

```diff
@@ -239,6 +239,16 @@ IIMethod::registerDisconElemFamilyForJumps(const unsigned int part)
 {
     TBOX_ASSERT(!d_fe_equation_systems_initialized);
     TBOX_ASSERT(part < d_num_parts);
+    if (d_velocity_jump_fe_family == LAGRANGE || d_pressure_jump_fe_family == LAGRANGE 
```

Comment:
If you need to set this in the input file, what is the job of the function `registerDisconElemFamilyForJumps`?